contents/teams/replay/objectives.mdx

```diff
@@ -1,14 +1,78 @@
-# 🚀 Goal 1: Get Alex 2 rocketing 2
+# 🚀 Goal 1: Keeping momentum on the things that are working
 
-it's great to have Alex 2 on the team... let's make it a success
+* Commenting on recordings (@pauldambra)
+    * Search by comments
+    * Emoji comment
+* Weekly digest (@pauldambra)
+    * Track retention of groups that see it vs don’t
+    * Talk to them and see what led them to click through
+* Playback UI (@veryayskiy)
+    * Mix-max
+    * Cinema mode
 
-# 👩‍🔬 Goal 2: Find the needle in the haystack
+# 👩‍🔬 Goal 2: Add things folk are asking for
 
-Many of our customers have lots of recordings, but struggle to find the "useful" ones. We want to test and iterate  on a bunch of different ideas to get customers to value faster.
+* Custom retention (Tue with Paul supporting)
+    * Longer retention
+        * This comes up often enough that it would be awesome to support it
+        * Plus the existing longer retention implementation is at best O(n)
+        * https://github.com/PostHog/product-internal/pull/766
+        * https://github.com/PostHog/product-internal/pull/738
+        * I think it would be a pretty powerful upsell
+    * Shorter retention
+        * some customers want to only keep recordings for 30 days (e.g. GDPR)
+        * not currently possible
+    * Some customers want to delete recordings **but not one a time**
+        * sometimes this is "delete all recordings because accidental PII" (we have break glass for this)
+        * sometimes this is "delete specific recordings because accidental PII" -> this is tricky right now
+        * sometimes this is "ha ha hah i have figured out a hack to get free recordings by deleting them before you charge me" -> user education needed
+   * activity log and access control
+        * all of these things need to go in the activity log to help track this down
+        * the new access control work might need to take it in to account
+* Conditional recording 3000 (@pauldambra)
+    * We’ve levelled up conditional recording but it could be much more cool
+    * E.g. sample 20% of mobile recordings but 60% of web recordings
+    * Or trigger with flag X at 20% sampling, event Y at 50% sampling, and on URL matching Z at 1% sampling
+* Working with customers who are building on top of us (@veryayskiy)
+    * Folk building on top of us is a new user group for us (and we don't measure their revenue separately 🤔)
+    * Currently they’re using our API which is a poor experience for everyone
+    * batch exports?
```

Comment:
```suggestion
    * batch exports
```
We should ship something here in Q3, if it's not batch exports, we should rephrase the heading.